### DISCUSSION TITLE
Cura 5.9 beta 2

### DISCUSSION THREAD

**Discussion by fieldOfView** [Ultimaker/Cura#19864] [Releases]
I have a question about the release notes of beta 2:

> This second beta introduces a change that affects the UI 

I can't find the commit that changes the UI. Since I maintain a couple of plugins that are tied very tightly to the UI, I would like to know more. Is it a change to the QML, to the underlying py modules, or an update to PyQt or Qt? I guess the latter would not show up in the cura repo but in one of the build repos.

Edit: and congratulations on the awesome printer name pun hidden in the screenshots.

**Comment by Rick-TRIGGER** [comment]
in the Mac version theres is a problem with the full Z plate lenght

**Comment by yidome** [comment]
cannot work on windows 8.1
![2024-11-25 21_47_10-WhatsApp - Opera](https://github.com/user-attachments/assets/baaccfc1-9c4d-420b-92be-2be51b51c572)

**Comment by Rick-TRIGGER** [comment]
Its not Windows 8.1.I talk about cura 8.1 on a Apple Mac.Eric Trudel450 521-9330 ***@***.*** 25 nov. 2024 09 h 50, yidome ***@***.***> a écrit :
cannot work on windows 8.1
2024-11-25.21_47_10-WhatsApp.-.Opera.png (view on web)

**Comment by Pojavlauncherrs** [comment]
If the UI changes in Cura 5.9 beta 2 end up causing conflicts with your plugins, you might also want to take a look at [PrusaSlicer](https://prusaslicers.com/). It’s fairly stable in terms of UI structure, and its open documentation makes it easier to keep plugins or custom workflows aligned.
